### ISSUE TITLE
How to display two view the same time by using SwiftEntry ?

### ISSUE THREAD

**Issue by Ssiswent** [huri000/SwiftEntryKit#340]
As the priority exists, if I display a new view above the view which has been displayed, the old one would be dismissed.

**Comment by huktoDev** [comment]
Yes, I too require ability to show few entries simultaneously. By example, popups and notifications. EKWindowProvider doesn't allow this, it contains only one entryWindow. EKWindowProvider should manage multiple windows, or we should have capability to use EKWindowProvider's directly

**Comment by purepure** [comment]
I love this feature too.

**Comment by wonder2011** [comment]
Nothing about this ?
